frontend/src/app/actions/experiments.js

```diff
@@ -4,22 +4,13 @@ export default createActions({
 
   updateExperiment: (addonID, data) => ({ addonID, data }),
 
-  fetchExperiments: (experimentsUrl, countsUrl) => {
-    let experiments = [];
-    return fetch(experimentsUrl)
-      .then(response => response.json())
-      .then(data => {
-        experiments = data.results;
-        return fetch(countsUrl);
-      })
-      .then(response => response.json())
+  fetchUserCounts: (countsUrl) => {
+    return fetch(countsUrl)
+      .then(
+        response => response.json(),
+        () => ({}))
```

Comment:
I dunno, error handling here feels excessive. If an error happens should we just not have the counts (like this) or fall back to something else (which can get complicated)? If we reported an error somewhere, who's going to look at it and be able to act on it? Also there's zero error handling in the current code either (not an excuse, but we're not having problems with it either)

That's why I went with 'meh'. If you have ideas for handling it I'm all ears.